### ISSUE TITLE
Dedicated Server

### ISSUE THREAD

**Issue by Etherealphenixx** [scfmod/FS22_TerraFarm#47]
This is just a little heads up. I know multiplayer may not be completely supported. So far it works fine in multiplayer but there must be a dedicated server side script or something causing an issue because it only digs gravel and never seems to get the buckets above 56% full but only when on a dedicated server. 

Absolutely love what you have accomplished so far. This post is not a “please fix this now” type of post. Just alerting of issues possibly not tested yet. Keep up the great work and have a great 2022.

**Comment by scfmod** [comment]
Yeah the current version have that problem due to not having proper MP sync of machines.
The new version does :-)

**Comment by gilbertwilli** [comment]
die große frage ist nur wann ist der release ? die aufgaben die für 4.0 notwendig sind sind anscheinend abgearbeitet wenn die infos im reiter projekt stimmen. wir warten so sehnsüchtig. der beste mod der je für ls22 raus kam.

**Comment by scfmod** [comment]
It's ready when it's ready :-) would love to have more spare time to work on it, but that's the way of life

**Comment by gilbertwilli** [comment]
sorry ich wollt nicht unhöflich sein

**Comment by scfmod** [comment]
No worries
